.github/workflows/auto-merge/automerge

```diff
@@ -84,22 +85,12 @@ def auto_merge(number, sha):
         print('SUCCESS - auto-merge')
         sys.exit(0)
     else:
-        comment(number, """**FAILURE** - Unable to auto-merge due to conflicts. Manual operation is required.
-
-To maintainers, please use the following steps to fix the merge conflicts manually:
-This is a example to fix conflict from `branch-0.2` to `branch-0.3`
-```bash
-git fetch <spark-rapids_remote>
-git checkout -b branch-0.3-merge-branch-0.2 <spark-rapids_remote>/branch-0.3
-git merge <spark-rapids_remote>/branch-0.2
-# fix the merge conflict, then
-git commit -am "merge branch-0.2 to branch-0.3"
-git push <your forked remote> branch-0.3-merge-branch-0.2
+        print('FAILURE - auto-merge')
+        comment(number=number, content=f"""**FAILURE** - Unable to auto-merge. Manual operation is required.
```

Comment:
Why were the instructions on how to manually resolve the auto-merge removed?

Reply:
we can hardly tell if it is merge conflict or other issues by status code, we have to add some regex for matching in message, but messages could be changed in the future. So I decided to replace that with the original failure info from response. 

And people should know how to fix the conflict I assume :)